src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -360,6 +376,30 @@ const formatAuthenticatorOptions = (
         AUTHENTICATOR_KEY.PHONE,
       ];
 
+      // @ts-ignore TODO: Add grace period fields to auth-js SDK https://oktainc.atlassian.net/browse/OKTA-848910
+      const date = new Date(authenticator?.gracePeriod?.expiry);
+      const gracePeriodEpochTimestampMs = !Number.isNaN(date.getTime()) ? date.getTime() : 0;
+      const currentTimestampMs = new Date().getTime();
```

Comment:
nit: `Date.getTime()`

Reply:
I think you meant `Date.now()` here? Updated to `Date.now()`

Reply:
> I think you meant Date.now() here? Updated to Date.now()

Oh yes, my mistake